sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubIO.java

```diff
@@ -723,84 +666,81 @@ public Duration getMaxReadTime() {
 
       private class PubsubReader extends DoFn<Void, T> {
```

Comment:
If I understand it correctly, looks like this is used for bounded PCollections only. Could we update the names for that?

BoundedPubsubReader, BoundedPubsubWriter?